manifest.json

```diff
@@ -6,22 +6,30 @@
         "en": "Virtual Private Networks (VPN) via WireGuard, with a web UI",
         "fr": "Réseaux Privés Virtuels (VPN) via WireGuard, avec une web UI"
     },
-    "version": "0.2.7~ynh1",
+    "version": "0.2.7~ynh2",
     "url": "https://github.com/ngoduykhanh/wireguard-ui",
     "license": "MIT",
     "maintainer": {
         "name": "tituspijean",
         "email": "tituspijean@outlook.com"
     },
     "requirements": {
-        "yunohost": ">= 3.8.1"
+        "yunohost": ">= 4.0.8"
     },
     "multi_instance": false,
     "services": [
         "nginx"
     ],
     "arguments": {
         "install" : [
+            {
+                "name": "warning",
+                "type": "display_text",
+                "ask": {
+                    "en": "WireGuard is still experimental. The WireGuard package for YunoHost will add a DMKS module to your Linux kernel.\n\nCheck its compatibility before installing it on a production system.",
+                    "fr": "WireGuard est encore expérimental. Le package WireGuard pour YunoHost va installer un module DKMS à votre kernel Linux.\n\nVérifiez sa compatibilité avant de l'installer sur un serveur de production."
```

Comment:
- Is the YunoHost package still experimental ? maybe because of Wireguard UI ?
- There is a need to tell about the DKMS module, but no need to worry the user about compatibility because this is a debian DKMS package, it's compatible with any debian kernel.

Reply:
* Yes, I am not 100% at ease with the WireGuard UI I chose. We could select another one later on.
* I meant to add that DKMS module may require to reboot the server.  Do you agree with that?

Reply:
Tell the user that a reboot is required is fine for me.